chrome.manifest

```diff
@@ -43,17 +43,19 @@ overlay chrome://messenger/content/messenger.xul  chrome://exchangecalendar/cont
 overlay chrome://messenger/content/folderProps.xul  chrome://exchangecalendar/content/folderPropsOverlay.xul
 overlay chrome://messenger/content/messenger.xul chrome://exchangecalendar/content/rtews-overlay.xul
 
-locale exchangecalendar nl locale/exchangecalendar/nl/
+
 locale exchangecalendar en locale/exchangecalendar/en/
-locale exchangecalendar en_US locale/exchangecalendar/en_US/
```

Comment:
Is the order of elements relevant? If not, I would suggest alphabetical ordering (applied on the entire line) so that future additions can always go to a deterministic location instead of an arbitrary location chosen by a developer.

Reply:
If I have well understand the manifest file, order for locale lines isn't relevant except for the first one which seems to be the default one used if the current locale hasn't any translation.